### PR TITLE
DOCSP-32840 1.2.0 release notes

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Relational Migrator 1.2.0 release notes

STAGING: https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-32840/release-notes/#1.2.0-changelog

JIRA: https://jira.mongodb.org/browse/DOCSP-32840

BUILD: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64f7613424fcc731b47d2ba3

### CODE REVIEW

**Review by tomhollander — APPROVED** [review]
Looks good!

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,27 @@ Release Notes
...
+
+Bug fixes:
+
+- Improvements to validation and error handling in mapping rules.
+- Ability to parse a wider range of Oracle DDL files.
+- Fixes an issue where tables were lost when refreshing a schema using SQL Server and Windows authentication. 
+- Fixes an issue where CDC updates may target the wrong document when tables do not have a primary key.
+- Fixes an issue where database passwords can be incorrectly saved to the browser's password library. 
+
+
```
Comment (ajhuh-mdb) [inline source/release-notes.txt:33]: [nit] looks like there's an extra newline here!

File: source/release-notes.txt [hunk 2]
```diff
@@ -10,6 +10,27 @@ Release Notes
...
+New features:
+
+- Ability to change the data type for migrated fields.
+- Ability to control how null values are represented in the migrated data.
+- Ability to use calculated fields for a collection's ``_id`` value.
+- Diagnostic dump includes configuration files to help streamline support. 
+
+Bug fixes:
+
+- Improvements to validation and error handling in mapping rules.
```
Comment (ajhuh-mdb) [inline source/release-notes.txt:27]: [suggestion] For parallelism, maybe we can start the bullets here to start with verbs and change this bullet to:
```suggestion
- Improves validation and error handling in mapping rules.
```

Comment (ajhuh-mdb) [reply source/release-notes.txt:27]: Similar suggestion for the bullet below!

**Review by ajhuh-mdb — APPROVED** [review]
LGTM with a few suggestions!

File: source/release-notes.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/release-notes.txt]: Done, thank you!

File: source/release-notes.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/release-notes.txt]: Done, thank you!

**Pull request merged**